app/views/dashboard/_tab_uploads.html.erb

```diff
@@ -21,25 +21,41 @@
 
```

Comment:
If you use my suggestions, you'll need to refactor the view accordingly, perhaps by breaking out the table row to its own partial and passing each set of orgs to the new partial.

Reply:
@corylown thanks for the review! I'll go with your suggestion. I guess I thought that info might be useful somewhere else in the app at some point to know which apps were inactive. But agree we can keep it simple for now.

Will address comments this afternoon.